### PR TITLE
IGVF-2030-remainin-enum-desc

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/construct_library_set.json [hunk 1]
```diff
@@ -709,6 +709,16 @@
                 "alleles",
                 "targeton"
             ],
+            "enum_descriptions": {
+                "tile": "The construct library was designed to target a tile of a protein region.",
```
Comment (hitz) [inline src/igvfd/schemas/construct_library_set.json:713]: "the construct library was designed to tile over a region of the genome or protein"

File: src/igvfd/schemas/crispr_modification.json [hunk 2]
```diff
@@ -122,6 +122,15 @@
                 "localizing",
                 "prime editing"
             ],
+            "enum_descriptions": {
+                "activation": "The CRISPR system is used to upregulate the expression of a target gene.",
+                "base editing": "The CRISPR system induces double-strand breaks at specific genomic locations to directly edit base sequences.",
```
Comment (hitz) [inline src/igvfd/schemas/crispr_modification.json:127]: "... changing one base pair to another"

File: src/igvfd/schemas/crispr_modification.json [hunk 3]
```diff
@@ -122,6 +122,15 @@
...
+                "cutting": "Involves using a catalytically inactive Cas protein to bind to specific DNA sequences and block transcription.",
+                "interference": "CRISPR technique aimed at completely disabling a gene's function.",
```
Comment (hitz) [inline src/igvfd/schemas/crispr_modification.json:129]: "...via recruiting silencing chromatin modifications"

File: src/igvfd/schemas/crispr_modification.json [hunk 4]
```diff
@@ -122,6 +122,15 @@
                 "localizing",
                 "prime editing"
             ],
+            "enum_descriptions": {
+                "activation": "The CRISPR system is used to upregulate the expression of a target gene.",
```
Comment (hitz) [inline src/igvfd/schemas/crispr_modification.json:126]: "...via recruiting activating chromatin modifications"

File: src/igvfd/schemas/crispr_modification.json [hunk 5]
```diff
@@ -122,6 +122,15 @@
...
+                "base editing": "The CRISPR system induces double-strand breaks at specific genomic locations to directly edit base sequences.",
+                "cutting": "Involves using a catalytically inactive Cas protein to bind to specific DNA sequences and block transcription.",
+                "interference": "CRISPR technique aimed at completely disabling a gene's function.",
+                "knockout": "Uses the CRISPR system to localize specific proteins or functional domains to particular genomic sites.",
```
Comment (hitz) [inline src/igvfd/schemas/crispr_modification.json:130]: I don't think that's right at all; doesn't it actually cut the whole gene out?  Or am I confusing this with interference?

File: src/igvfd/schemas/prediction_set.json [hunk 6]
```diff
@@ -244,13 +244,18 @@
...
             "title": "Prediction Scope",
-            "description": "The scope or scale that this prediction set is designed to target. If the scope is across gene(s) or loci, these will need to be specified in the genes or loci property.",
+            "description": "The scope or scale that this prediction set is designed to target.",
             "type": "string",
             "enum": [
                 "genes",
                 "loci",
                 "genome-wide"
             ],
+            "enum_descriptions": {
```
Comment (hitz) [inline src/igvfd/schemas/prediction_set.json:254]: the use of "upon" here is weird but I can't come up with anything better at the moment.

File: src/igvfd/schemas/treatment.json [hunk 7]
```diff
@@ -311,7 +312,17 @@
                 "selection",
                 "stimulation"
             ],
-            "description": "The intended purpose for treating the samples; Activation: treatment is known to activate a pathway in the biosample; Agonist: a substance which is known to initiate a physiological response when combined with a receptor; Antagonist: a substance that is known to interfere with or inhibits the physiological action of another; Control: treatment applied to a sample for control purposes; Differentiation: treatment that is applied to convert a less specialized cell to a more specialized cell; De-differentiation: treatment used to reprogram differentiated cells back to less determined cell states; Perturbation: treatment applied to the sample in order to study the effect of its application; Selection: treatment used to affect biosample in a way that can be used to distinguish cells and select for in the downstream steps; Stimulation: treatment applied to stimulate a cellular pathway.",
+            "enum_descriptions": {
+                "activation": "Treatment is known to activate a pathway in the biosample.",
```
Comment (hitz) [inline src/igvfd/schemas/treatment.json:316]: "...activate a pathway or modification in the biosample".

**Review by hitz — CHANGES_REQUESTED** [review]
mostly good just some minor corrections

File: src/igvfd/schemas/construct_library_set.json (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/construct_library_set.json:713]: fixed

File: src/igvfd/schemas/crispr_modification.json (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/schemas/crispr_modification.json:127]: fixed

File: src/igvfd/schemas/crispr_modification.json (hunk 3 above)
Comment (ian-whaling) [reply src/igvfd/schemas/crispr_modification.json:129]: fixed

File: src/igvfd/schemas/crispr_modification.json (hunk 4 above)
Comment (ian-whaling) [reply src/igvfd/schemas/crispr_modification.json:126]: fixed

File: src/igvfd/schemas/treatment.json (hunk 7 above)
Comment (ian-whaling) [reply src/igvfd/schemas/treatment.json:316]: fixed

**Review by hitz — APPROVED** [review]

**Pull request merged**